### DISCUSSION TITLE
Add the  rule missing propiety(Query or body)

### DISCUSSION THREAD

**Discussion by hitokiri** [mockoon/mockoon#491] [Ideas] [answered]
Add one rule for when in the body or the query does not have one property.

need:
{
    "professional": true,
    "veteran": false,
    "firstName": "null",
    "lastName": "lol",
    "email": "01Test@01test.com",
    "zip": "90210",
    "phone": "123456",
    "age": "36",    
    "registeredApprentice": true,
    "constructionTrade": [],
}

I receive this (missing the professional ):
{
    "veteran": false,
    "firstName": "null",
    "lastName": "lol",
    "email": "01Test@01test.com",
    "zip": "90210",
    "phone": "123456",
    "age": "36",    
    "registeredApprentice": true,
    "constructionTrade": [],
}

**Comment by 255kb** [comment, marked as answer]
Thank you for your suggestion, there is already an open issue for this #255. This would be indeed a useful enhancement.
